run_route_scripts/run_with_server.py

```diff
@@ -21,34 +21,54 @@ def get_post_bodies(filename):
       post_body['id'] = str(line_number)
       yield post_body
 
-# for persistant connections
-def initialize(url_, output_dir_):
+def initialize(args_):
```

Comment:
i did a bunch of refactoring here to make this script more useful. first i moved all the program arguments to a global so that all processes could see them all. then i added the null format option which is useful when you just want to performance test an endpoint but dont care to parse the result (isochrone diffing makes no sense tbh)